packages/story-editor/src/app/highlights/quickActions/useQuickActions.js

```diff
@@ -214,17 +214,8 @@ const useQuickActions = () => {
   );
 
   const handleFocusMediaPanel = useMemo(() => {
-    const idOrigin = selectedElements?.[0]?.resource?.id
-      ?.toString()
-      .split(':')?.[0];
-    const is3PGif =
-      (!idOrigin || idOrigin?.toLowerCase() === 'media/tenor') &&
-      selectedElements?.[0]?.resource?.type?.toLowerCase() === 'gif';
-    const is3PVideo = idOrigin?.toLowerCase() === 'media/coverr';
-    const is3PImage = idOrigin?.toLowerCase() === 'media/unsplash';
-
-    const panelToFocus =
-      is3PImage || is3PVideo || is3PGif ? states.MEDIA3P : states.MEDIA;
+    const isExternal = selectedElements?.[0]?.resource?.isExternal;
+    const panelToFocus = isExternal ? states.MEDIA3P : states.MEDIA;
```

Comment:
Interesting, didn't know about this code.

Looks like it's used to highlight the Media3p tab when using quick actions on a media3p image/video.

The thing is, with this change it will also the highlight the Media3p tab for hotlinked media, which is incorrect.

So this change should be reverted or perhaps replaced by a new `isMedia3p` flag on resources (in addition to `isExternal`)

Reply:
An alternative would be to simplify this check like this:

```js
const is3pMedia = resourceId.startsWith('media/');
const panelToFocus = is3pMedia ? states.MEDIA3P : states.MEDIA;
```